Cargo.toml

```diff
@@ -6,7 +6,7 @@ authors = ["soywod <clement.douin@posteo.net>"]
 edition = "2018"
 
 [dependencies]
-clap = "2.33.3"
+clap = {version = "2.33.3", default-features = false, features=["suggestions"]}
```

Comment:
By doing so, you also remove colors in all help menus since it's part of the [default features](https://github.com/clap-rs/clap#features-enabled-by-default). I think we should put back the "color" feature at least. For now I manage colors myself, which is sth to avoid. I should use the same [libs](https://github.com/clap-rs/clap/blob/8b721a2bcdcf412c67342a6bdf2d540150dbd37e/Cargo.toml#L92) as clap.

Reply:
Added!